src/Basic/NaturalTransformation.lidr

```diff
@@ -159,3 +160,72 @@ The code above is everything we need to define what a natural transformation is.
 >                             (mapObj fun2 b)
 >                             (mapMor fun1 a b f)
 >                             (component b)
+>
+> naturalTransformationExt :
+>   (cat1, cat2 : Category)
```

Comment:
usually we indent this line so that the parenthesis is aligned with the one in the next line. Plead adjust here and on the rest of the code